CONTRIBUTING.md

```diff
@@ -87,36 +88,41 @@ Please add the following line to your settings.json file :
     "go.testTags": "unit,e2e"
 ```
 
-This will enable compilation for unit test and end-to-end tests.
+This will enable compilation for unit and end-to-end tests.
 
 #### Debugging in VSCode
 
-To debut in VSCode you need to create a debug configuration for the command with required arguments.
-Run the following commands to 
+To debug in VSCode, you need to create a debug configuration for the command with required arguments.
+Run the following commands to create a new launch.json file for the debugger:
 
 ```
 touch .vscode/launch.json
 ```
-Then put the following configuration into the file.
-Review and replace command name and arguments depending on the command you are using.
+Then put the following example configuration into the file.
+Review and replace the command name and arguments depending on the command you wish to debug.
 
 ```json
 {
     "configurations": [
         {
-            "name": "Login Command",
+            "name": "DBuser Create Command",
             "type": "go",
             "request": "launch",
             "mode": "auto",
             "program": "${workspaceFolder}/cmd/atlas",
             "env": {},
             "args": [
-              "login"
+              "dbuser",
+              "create",
+              "atlasAdmin", 
+              "--username",
+              "myUser",
+              "--password",
+              "123abc"
```

Comment:
I suggest changing the example configuration to something with more args and flags required to more clearly show how the configuration needs to be altered for alternate commands.